python/plugins/processing/gui/ConfigDialog.py

```diff
@@ -81,6 +81,9 @@ def __init__(self, parent):
     def apply(self):
         self.config_widget.accept()
 
+    def helpKey(self):
+        return 'processing/toolbox.html#configuring-the-processing-framework'
```

Comment:
I can't tell about the code logic but while the link used above sounds the most appropriate we (I) currently back to the [chapter level](http://docs.qgis.org/testing/en/docs/user_manual/processing/index.html) (see e.g. http://docs.qgis.org/testing/en/docs/user_manual/introduction/qgis_configuration.html#processing ) because i do not feel comfortable with the structure of this chapter (eg, configs should imho not be sub-item of toolbox section; they deserve their own section and there are some other redundancies)? Moving in the future the config section will break your link while the global chapter link sounds always safer (despite its inaccuracy).

Reply:
So which key should we be using here?

Reply:
Let's use "processing/index.html" for the moment. 
Sorry for having not been clear.